### DISCUSSION TITLE
请问下怎么给设计器自增表格配置默认数据？

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2540] [Q&A 2.x] [answered]
如下配置并不可行，请问需要如何配置默认表格数据，没找到相关配置文档，求帮助！
![image](https://user-images.githubusercontent.com/37055241/144188474-1c93959c-4146-41dd-aa21-178329841aed.png)
![image](https://user-images.githubusercontent.com/37055241/144188475-c894ee22-97ac-489b-ba5c-5de37abf7f6b.png)

**Comment by huaxiangsirenbutong** [comment]
@cncolder @s0ngyee 求解答

**Comment by janryWang** [comment, marked as answer]
默认值配置表达式，传数组
